src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/EnvironmentSetupHelper.cs

```diff
@@ -369,7 +369,12 @@ public virtual Collection<PSObject> RunPowerShellTest(params string[] scripts)
                     if (powershell.Streams.Error.Count > 0)
                     {
                         throw new RuntimeException(
-                            "Test failed due to a non-empty error stream, check the error stream in the test log for more details.");
+                            string.Format(
+                                "Test failed due to a non-empty error stream. First error: {0}{1}",
+                                PowerShellUtilities.FormatErrorRecord(powershell.Streams.Error[0]),
```

Comment:
It's used in the next file https://github.com/Azure/azure-powershell/pull/3663/files#diff-90463d182d91894d58358adc66b14651R119